pkg/metrics/metrics.go

```diff
@@ -46,6 +48,8 @@ const (
 	AssignedIdentityDeletionOperationName = "assigned_identity_deletion"
 	// AssignedIdentityAdditionOperationName ...
 	AssignedIdentityAdditionOperationName = "assigned_identity_addition"
+	// AssignedIdentityUpdateOperationName ...
```

Comment:
Let's open another PR to clean up these `...` comments as I noticed there are a lot in the codebase.

Reply:
Opened an issue to track this - https://github.com/Azure/aad-pod-identity/issues/571. We can use this as a parent issue and add sub items as we make the changes.